Ambient transactions are supported in Core 2.1 (#4215)
Particular/docs.particular.net: transports/sql/transactions_ambient-core-warning_sqltransport_[4,).partial.md

#### Distributed transactions are not supported in .NET Core

> Although the .NET Core SQL Server driver supports the ambient transactions, it does not support the Distributed Transaction Coordinator so any attempt to use more than one transactional resource within an ambient transaction will cause an error.

Reviewer: ```suggestion
Although the .NET Core SQL Server driver supports enlisting in ambient transactions, it does not support participating in distributed transactions so any attempt to use more than one transactional resource within the same ambient transaction will cause an error.
```

> Prior to Version 2.1, the .NET Core SQL Server driver did not support the ambient transactions and any attempt to use it resulted in the following exception:

Reviewer: ```suggestion
Prior to Version 2.1, the .NET Core SQL Server driver did not support enlisting in ambient transactions and any attempt to do so resulted in the following exception:
```
